src/ResourceManager/RecoveryServices.SiteRecovery/AzureRM.RecoveryServices.SiteRecovery.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.1.6'
+ModuleVersion = '0.1.5'
```

Comment:
@viverm looks like a bad merge caused some unnecessary changes in this file

Reply:
Changed to 0.1.6